### PR TITLE
update for polymer 1.0-rc1 (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by jakemac53**
**This version of polymer is not yet released, so please wait to merge**

Fixes https://github.com/dart-lang/polymer-dart/issues/552

### CODE REVIEW

**Comment by devoncarew** [comment]
cc @kwalrath, @Sfshaza

File: templates/web-polymer/pubspec.yaml [hunk 1]
```diff
@@ -8,10 +8,20 @@ environment:
   sdk: '>=1.0.0 <2.0.0'
```
Comment (donny-dont) [inline templates/web-polymer/pubspec.yaml]: Shouldn't the SDK dependency be bumped to `'>=1.9.0 < 2.0.0'` if you want to use the `^` in your dependencies?

Comment (Sfshaza) [inline templates/web-polymer/pubspec.yaml]: Good catch! Caret syntax was added in 1.8.3, so 1.9.0 seems reasonable.

**Comment by devoncarew** [comment]
lgtm!

**Comment by jakemac53** [comment]
thanks!

**Pull request merged**